NetObserv FlowMetric API regeneration (#83184)
openshift/openshift-docs: modules/network-observability-flowmetric-api-specifications.adoc

| `string`
| Name of the metric. In Prometheus, it is automatically prefixed with "netobserv_".

| `remap`
| `object (string)`

> | `remap` allows to use different names for the generated metric labels than the flow fields. Use the origin flow fields as keys, and the desired label names as values.

Reviewer: ```suggestion
| Set the `remap` property to use different names for the generated metric labels than the flow fields. Use the origin flow fields as keys, and the label names that you want as values.
```
- I reworded this line a little to avoid anthropomorphic language ("allows").
- IBM Style word usage warns against using "desired" in technical content.
